### PR TITLE
Moving the summary message functions from the app to the api router

### PR DESCRIPTION
**Pull request by mairlot**
Started turning the scripts into endpoints, just the summary one as a first go. Thinking about adding most of the slack configuration part as body in the post request so it can be controlled from there rather than having to make changes in the code. Let me know what you think 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,3 +12,19 @@ Where to find a messages messageTsId - Do inspect element on the slack web app
...
+All endpoints are under the prefix of `/kudos`.
+
+##### Summary - `/summary`
+
+This sends the summary message of multiple kudos messages. TODO Example body:
+```json
+{
+    "from": 7,
+    "to": 0,
+    "channel": "GENERAL",
```
Comment (tonyhkly) [inline README.md:27]: Really cool!

Comment (tonyhkly) [reply README.md:27]: Think the `from` and `to` params are a little confusing but no more than when they were constants

File: src/server/modules/summary/index.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        console.log(`Got ${kudosMessages.length} thank you messages`);
+
+        if (kudosMessages.length) {
+            await sendKudosMessages(app, kudosMessages, SEND_TO_CHANNEL_ID)
+        }
+
+        res.status(200).send();
+    } catch (e) {
+        console.log(e);
+        res.status(500).send(e.toString());
```
Comment (tonyhkly) [inline src/server/modules/summary/index.js:20]: We're exposing the error message to a FE api 😱 . I feel like we should have a tech design meeting to discuss this approach in lengthy detail

**Review by tonyhkly — APPROVED** [review]
This is cool! One thing I'm realising with github reviews is how hard it is to see file structure compared to butbucket. `The Jump to...` is ok but not that great

File: README.md (hunk 1 above)
Comment (mairlot) [reply README.md:27]: Yeah, totally agree about the from and to params, think best plan would be is to make them date strings `YYYY-MM-DD`

**Pull request merged**